doc/api/crypto.md

```diff
@@ -1320,7 +1316,7 @@ console.log(alice_secret == bob_secret);
 added: v0.9.3
 -->
 
-Returns an array with the names of the supported hash algorithms.
+Returns an array with the names of the supported hash algorithms. For example, `RSA-SHA256`.
```

Comment:
The added text is a sentence fragment (which is fine for the first "sentence", but subsequent statements should be complete sentences). It seems unnecessary anyway. If we *do* want to include the example, I'd prefer:

> Returns an array with the names of the supported hash algorithms, such as `RSA-SHA256`.